### PR TITLE
Access applicant cv

### PR DESCRIPTION
**Pull request by GioGiunta2606**

### CODE REVIEW

File: client/src/App.css
Comment (filippoCaste) [inline client/src/App.css:1]: This file has been removed because it wasn't used right?

File: server/database.sqlite
Comment (filippoCaste) [inline server/database.sqlite:1]: Do not merge this db with the one in the main

File: server/src/config/db.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import sqlite from "sqlite3";
 
 // open the database
-const db = new sqlite.Database("database.db", (err) => {
+const db = new sqlite.Database("database.sqlite", (err) => {
```
Comment (filippoCaste) [inline server/src/config/db.js:4]: do not merge this in the main

File: server/src/config/configs.js [hunk 2]
```diff
@@ -72,4 +48,35 @@ export const isTeacher = async (req, res, next) => {
     }
   }
   return res.status(401).json({ error: "Not authorized" });
-};
\ No newline at end of file
+};
+
+const storage = multer.diskStorage({
+  destination: (req, file, cb) => {
+    const uploadPath = path.join(__dirname, '../../public/students_CV');
```
Comment (filippoCaste) [inline server/src/config/configs.js:55]: if i am not wrong, the folder public is not okay. It should be another one.
I will provide you more info in following comments.

File: server/src/services/career.services.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+            return reject(err);
+        }
+        resolve(rows);
+        });
+    });
+};
+
+export const getStudentCV = (studentId, applicationId) => {
+  const fileName = `s${studentId}_${applicationId}_CV.pdf`;
+  const filesFolder = path.join(process.cwd(), 'public', 'students_CV');
```
Comment (filippoCaste) [inline server/src/services/career.services.js:19]: in case the public folder is not okay, also here should be changed

File: client/src/App.css
Comment (GioGiunta2606) [reply client/src/App.css:1]: Yes

File: server/database.sqlite
Comment (GioGiunta2606) [reply server/database.sqlite:1]: I forgot to change it

File: client/src/components/ApplicationDialog.jsx [hunk 4]
```diff
@@ -1,24 +1,60 @@
...
+      {renderField("Id", student_id)}
+      {renderField("Email", student_email)}
+      {renderField("Title Degree", student_title_degree)}
+      {renderField("Enrollment Year", student_enrollment_year)}
+      {renderField("Nationality", student_nationality)}
+      {renderField("Submission Date", dayjs(submission_date).format("DD/MM/YYYY"))}
+      {renderField("Status", status)}
+    </>
+  );
+};
```
Comment (s312121) [inline client/src/components/ApplicationDialog.jsx:46]: The user interface of this section can be improved similar to the responsive table that we implemented in AlertDialog component.

**Review by s312121 — APPROVED** [review]
It was a good pull request and apart from the implementation of the student's CV, it also resolved many of the technical issues and previous problems.

**Pull request merged**